src/challenges.js

```diff
@@ -19,11 +27,27 @@ const counter = () => {
   // Example: const newCounter = counter();
   // newCounter(); // 1
   // newCounter(); // 2
+  let count = 0;
+  function increment() {
+    ++count;
+    return count;
+  }
+  return increment;
 };
 
 const limitFunctionCallCount = (cb, n) => {
   // Should return a function that invokes `cb`.
   // The returned function should only allow `cb` to be invoked `n` times.
+  let count = 0;
+  const limiter = (...args) => {
+    if (count < n) {
+      count++;
+      return cb(...args) 
+  } else {
```

Comment:
There is a bit of formatting issues - there is a shortcut in formatting .js files on sublime!